packages/amplify-e2e-core/src/utils/projectMeta.ts

```diff
@@ -28,6 +28,9 @@ function getProjectMeta(projectRoot: string) {
   const metaFilePath: string = path.join(projectRoot, 'amplify', '#current-cloud-backend', 'amplify-meta.json');
   return JSON.parse(fs.readFileSync(metaFilePath, 'utf8'));
 }
+function getCustomPoliciesPath(projectRoot: string, category: string, resourceName: string): string {
```

Comment:
Can this be just used from pathmanager in core?

Reply:
No, it failed

Reply:
Ok I will have to take a look at it